### DISCUSSION TITLE
Decouple NiceGUI and Tailwind

### DISCUSSION THREAD

**Discussion by ghost** [zauberzeug/nicegui#2121] [Ideas / Feature Requests]
Would it be possible to decouple the CSS styling, in such a way that we could specify any other CSS framework, instead of Tailwind?

I am loath to hardwire CSS styling directly into an element and would prefer to link a stylesheet instead (ex.: Bootstrap instead of Tailwind). Most elements might require that you specify a style for the component, but this is preferable because one can easily override the CSS  by including an additional stylesheet.

**Comment by falkoschindler** [comment]
You can remove the Tailwind library via `ui.run(tailwind=False)`. This should leave you with nothing but some default CSS (and Quasar). Is that what you're trying to do?

**Comment by shirooo39** [comment]
stupid question but can I not use Quasar in place of something I choose myself? for example, [this Material Design 3 framework by a Chinese developer](https://github.com/zdhxiong/mdui)  
since there's `ui.add_head_html()`, maybe it's possible to import different framework and use the components through `ui.element()`?

**Reply by rodja** [reply]
Yes, you can use other libraries in addition to Quasar that way.
